src/types.ts

```diff
@@ -8,6 +8,10 @@ export interface HipWorkResponse<ResponseShape> {
   status?: number;
 }
 
```

Comment:
hmm... I wasn't really ready to decide whether initPreContext needs to be optional or not...

But now that executeHip is written, I think it should be optional.  We should make an issue to make sure it's consistently optional everywhere like the sanitizers.
- [ ] Make it optional for this PR
- [ ] FUI for making that consistent

Reply:
done
https://github.com/captaincaius/hipthrusts/issues/21#issuecomment-613550846